mir-ci/mir_ci/benchmarker.py

```diff
@@ -98,19 +99,19 @@ def _on_packet(self, packet: ProcessInfoFrame) -> None:
         cpu_time_microseconds_total = packet.cpu_time_microseconds_total
         current_memory_bytes = packet.current_memory_bytes
         if pid is None:
-            print("Frame is lacking pid")
+            logging.getLogger().error("Frame is lacking pid")
```

Comment:
```suggestion
            logger.error("Frame is lacking pid")
```

And all the way down, of course.